barcodes_gs1_label/security/barcodes_gs1_label.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
As we talked several times, the filename should follow the OCA guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst (with the modifications in this specific case we already discussed)
So instead of `barcodes_gs1_label/security/barcodes_gs1_label.xml` should be `barcodes_gs1_label/security/options_config_security.xml`
Please don't forget that kind of things...